django_celery_results/models.py

```diff
@@ -143,9 +144,71 @@ def group_result(self, app=None):
             app (Celery): app instance to create the GroupResult with.
 
         """
-        return GroupResult(
+        return CeleryGroupResult(
             self.group_id,
             [result_from_tuple(r, app=app)
              for r in json.loads(self.sub_tasks)],
-            app=app,
```

Comment:
can you make sure removing this won't create any regression or backward compat issue?

Reply:
Im honestly not sure why I removed this line, we still allow the app to be passed in as a kwarg. I'm going to add it back